Twitter.js

```diff
@@ -12,5 +12,8 @@ module.exports = {
   },
   logOut: function () {
     SMXTwitter.logOut();
+  },
+  composeTweet: function (options, cb) {
+    SMXTwitter.sharePost(options, cb);
```

Comment:
@ramilushev Shouldn't this be `SMXTwitter.composeTweet(options, cb);` vs `SMXTwitter.sharePost(options, cb);`?